frontend/lib/services/api_services/placement_api_service.dart

```diff
@@ -9,7 +9,7 @@ class PlacementAPIService extends APIInfo {
     switch (subRoute) {
       case "/placement/new-placement":
         return _postPlacement(subRoute, body);
-      case "placements":
+      case "/placement":
```

Comment:
@AliceCasali why does this route exist? Why has it been implemented here?

Reply:
It was already there when I started working, then I've used it as a temporary route. I can delete it if you want, but maybe can be useful in the future

Reply:
The private function that it is calling is just a copy paste of the one above it, so I think that for the code's quality's sake we can delete it, it won't need any significant time effort in the future to re implement it.